catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/CopyFilterDelegate.java

```diff
@@ -100,6 +100,12 @@ public Filter propertyIsEqualTo(String propertyName, String literal, boolean isC
     }
 
     @Override
+    public Filter propertyIsEqualTo(String functionName, List<Object> arguments, Object literal) {
+        throw new UnsupportedOperationException(
+                "propertyIsEqualTo(String,List,Object) not supported by CopyFilterDelegate.");
```

Comment:
okay going to do the same thing i did for CswFilterDelegate. I will assume the first arg is a attributeName and the rest are values. Even if it isn't correct for all function it should still work since the filter adapter should still be able to convert to the correct object.